compose/backend.yml

```diff
@@ -87,52 +87,25 @@ services:
       - ccd-shared-database
       - ccd-user-profile-api
       - ccd-definition-store-api
-      - idam-api
+      - sidam-api
       - service-auth-provider-api
 
-  idam-api:
-    image: hmcts.azurecr.io/hmcts/tidam-api:latest
-    healthcheck:
-      interval: 10s
-      timeout: 10s
-      retries: 10
+  sidam-api:
+    image: hmcts.azurecr.io/hmcts/sidam/idam-api:1.0.0
     environment:
-      IDAM_SUPERUSER_EMAIL: ccd@hmcts.net
-      SPRING_DATASOURCE_URL: jdbc:postgresql://ccd-shared-database:5432/idam
-      SPRING_DATASOURCE_USERNAME: "${DB_USERNAME}"
-      SPRING_DATASOURCE_PASSWORD: "${DB_PASSWORD}"
-      SPRING_MAIL_HOST: smtp-server
-      IDAM_TESTING_SUPPORT_ENABLED: "true"
-      NOTIFY_API_KEY: AAAAAAAAAAAAAAAA
-      NOTIFY_CMC_ACTIVATE_USER_TEMPLATE: 76aa8695-64e8-4afd-ae13-bc8385302b1f
-      IDAM_SECRET: idam_test_secret
-      IDAM_API_OAUTH2_CLIENT_CLIENT_SECRETS_CCD_GATEWAY: "${OAUTH2_CLIENT_CCD_GATEWAY}"
-      IDAM_API_OAUTH2_CLIENT_CLIENT_SECRETS_CCD_ADMIN: "${OAUTH2_CLIENT_CCD_ADMIN}"
-      IDAM_CCD_WHITELIST: http://localhost:3451/oauth2redirect
-      IDAM_CCD_ADMIN_WHITELIST: https://localhost:3100/oauth2redirect
-      NOTIFY: "false"
-      NOTIFY_CMC_WELCOME_USER_TEMPLATE: fake
-      NOTIFY_DIVORCE_WELCOME_USER_TEMPLATE: fake
-      NOTIFY_SSCS_WELCOME_USER_TEMPLATE: fake
-      NOTIFY_RESET_PASSWORD_TEMPLATE: fake
-      NOTIFY_PROBATE_ACTIVATE_USER_TEMPLATE: fake
-      NOTIFY_DIVORCE_ACTIVATE_USER_TEMPLATE: fake
-      NOTIFY_SSCS_ACTIVATE_USER_TEMPLATE: fake
-      NOTIFY_CCD_ACTIVATE_USER_TEMPLATE: fake
-      REFORM_SERVICE_NAME: idam-api
-      REFORM_TEAM: idam
-      REFORM_ENVIRONMENT: local
-      http_proxy:
-      https_proxy:
-      no_proxy:
+      JAVA_OPTS: -Xms128m -Xms512m -Dlogging.level.uk.gov.hmcts=DEBUG
+      SPRING_DATASOURCE_URL: jdbc:postgresql://shared-db:5432/openidm?currentSchema=fridam
+      TESTING_SUPPORT_ENABLED: "true"
     ports:
-      - 4501:8080
+      - 5000:5000
```

Comment:
May i know the issue mapping with 4501 pls? (like 4501:5000). Otherwise we may have to change all other repos config / readme as they assumed idam is available on http://localhost:4501

Reply:
This was done as per the confluence page for local docker setup for sidam.